editor/src/components/navigator/left-pane/pages-pane.tsx

```diff
@@ -608,18 +611,6 @@ export const AddPageContextMenu = React.memo(
   },
 )
 
-function useScrollToNavigateToRoute(
```

Comment:
oh sorry I wanted to ping you. it was buggy and then while I was fixing it I realized that we don't want scrolling because the Favorites and the Routes are fighting with each other. maybe we can revisit it later but with a more complex ruleset (ie if you were clicking on a favorite, don't jump to Route). in general I think we should prefer scrolling to the favorites, because the new routes section is comprehensive but less interesting than it used to be

Reply:
gotcha! fine by me